airbyte-integrations/connectors/source-close-com/source_close_com/single_stream_slicer.py

```diff
@@ -0,0 +1,90 @@
+import datetime
+import pendulum
+
+from dataclasses import dataclass, field
+from typing import Optional, Iterable, Mapping, Any, Union
+
+from airbyte_cdk.models import SyncMode
+from airbyte_cdk.sources.declarative.datetime import MinMaxDatetime
+from airbyte_cdk.sources.declarative.interpolation import InterpolatedString
+from airbyte_cdk.sources.declarative.requesters.request_option import RequestOptionType, RequestOption
+from airbyte_cdk.sources.declarative.stream_slicers import SingleSlice
+from airbyte_cdk.sources.declarative.types import StreamState, StreamSlice, Record, Config
+
+
+@dataclass
+class SingleStreamSlicer(SingleSlice):
```

Comment:
is there a benefit to using a single slice instead of chunking the time range like DatetimeStreamSlicer does? Using multiple slices is nice because it gives us checkpointing for free

Reply:
There are streams that require a very small step, a millisecond or less. It looks like I cannot avoid customization

Reply:
Fine with me if it has to be custom, but I'd like to better understand the limitation of the implementation in the CDK.

The DatetimeStreamSlicer's `update_cursor` method also updates the cursor based on the last record read, so I would expect it to fetch the same records even if they are in different slices

Reply:
Figured out how to avoid custom Slicer, so custom component is removed